Files/Views/SettingsPages/Widgets.xaml

```diff
@@ -43,6 +43,13 @@
                     HeaderTemplate="{StaticResource CustomHeaderStyle}"
                     IsOn="{Binding ShowDrivesWidget, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
 
+                <ToggleSwitch
+                    x:Uid="SettingsWidgetsShowBundles"
+                    AutomationProperties.Name="Show Bundles on the home page"
```

Comment:
AutomationProperties shouldn't be needed since the string is the same as the header.

Reply:
I decided to keep it since other ToggleSwitches have the same property

Reply:
@d2dyno1 It might not be necessary for some of those toggle switches either, there is no need to add something extra because it's done that way somewhere else 😂